internal/tfimport/importer/resources.go

```diff
@@ -28,3 +49,81 @@ func fromConfigValues(key string, cvs ...ProviderConfigMap) (interface{}, error)
 	}
 	return nil, fmt.Errorf("could not find key %q in resource change or provider config", key)
 }
+
+// userValue asks the user to fill in a value that the importer can't figure out.
+func fromUser(in io.Reader, fieldName string, prompt string, choices []string) (val string, err error) {
+	log.Printf("Could not determine %q automatically\n", fieldName)
+	log.Println(prompt)
+
+	if len(choices) > 0 {
```

Comment:
Why do we need logic to pick from choices? random_id can't be given a choice. Can we add this only when it's actually needed?

Reply:
I was hoping to have a single function to call for "get value from user". I'll try to rearrange the code so you don't always have to pass in choices even if you're not using them.

Reply:
Rearranged, PTAL.